content/4_Gold/Combinatorics.mdx

```diff
@@ -807,6 +807,101 @@ print()
 </PySection>
 </LanguageSection>
 
+## Catalan Numbers
+
+The [Catalan numbers](https://en.wikipedia.org/wiki/Catalan_number) are a sequence of positive integers that can be very useful in counting problems in combinatorics. Some of the well-known applications are: count certain types of lattice path in a grid, counting binary trees, number of valid paranthesis strings of length $n$, and many more. The $n$-th Catalan can be expressed as following using binomial coeffiecients:
```

Comment:
one resource i think is great as well is the one by cp-algo. maybe also link it :D

https://cp-algorithms.com/combinatorics/catalan-numbers.html

Reply:
Done!